cli/pom.xml

```diff
@@ -192,25 +194,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>com.oneops.maven.plugins</groupId>
-                <artifactId>really-executable-jar-maven-plugin</artifactId>
-                <version>1.4.2</version>
-                <executions>
-                    <execution>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>really-executable-jar</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <flags>-Xmx128m -client</flags>
-                    <classifier>uber</classifier>
-                    <outputClassifier>executable</outputClassifier>
-                    <allowOtherTypes>true</allowOtherTypes>
-                </configuration>
-            </plugin>
```

Comment:
Any particular reason this is removed?

Reply:
> Also, the artifact produced by really-executable-jar-maven-plugin is being rejected by the central portal for whatever reason. Disabling it for now, will work on a workaround/replacement in a patch release.

Reply:
Something-something, unknown artifact type.

Reply:
ah dang, jumped into the changes first.